src/SearchField/SearchFieldAdvanced.jsx

```diff
@@ -100,7 +100,7 @@ function SearchFieldAdvanced(props) {
         role="search"
         onSubmit={handleSubmit}
         onReset={handleClear}
-        className="d-flex align-items-center w-100"
+        className="pgn__searchfield-form d-flex align-items-center w-100"
```

Comment:
Can you please also add styles from `d-flex align-items-center w-100` classes to the `pgn__searchfield-form` class that you created and only use it here?

We shouldn't be using these utility classes in our components because they use `!important` flag which makes it hard for consumers to override these styles (when you move these style to your new class, don't use `!important` flag there).

Reply:
I agree on this point

Reply:
yeah, sure